dcos/model_iam_error_code.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * DC/OS
+ *
+ * DC/OS API
+ *
+ * API version: 1.0.0
+ */
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package dcos
+
+type IamErrorCode string
+
+// List of IAMErrorCode
+const (
+	ACL_EXISTS                     IamErrorCode = "ERR_ACL_EXISTS"
```

Comment:
@bamarni @ArmandGrillet IMO the constants should have a prefix. Any idea how we could do it?

so it should be `IAM_ACL_EXISTS` theoretically we could have the same error in a different enum

Reply:
The solution you have implemented in your second commit looks good to me :+1:

Reply:
yeah I think that's what we use in another place